Pipfile

```diff
@@ -11,6 +11,8 @@ pandas = ">=0.24.2"
 gamebench-api-client = {editable = true,path = "."}
 coverage = ">=4.5.3"
 recommonmark = "*"
+gamebench-api-client-big-fish = {editable = true,path = "."}
+urllib3 = "*"
```

Comment:
Rather than "*" I think we need something like: urlib ~= 1.24.2 to say it must be 1.24.2 or later but not 2.0 or later.

Reply:
You are right. I updated it to be ">=1.24.2,<1.25". The requests library has a dependency on this where urllib3<1.25, so we encounter errors while it is greater than or within the same major revision.